scripts/download-CanBridge.mjs

```diff
@@ -2,47 +2,127 @@ import * as fs from "fs";
 import * as path from "path";
 import axios from 'axios';
 import AdmZip from 'adm-zip';
+import { platform, arch } from 'os';
 
-const canBridgeTag = "v2.5.1";
+const canBridgeTag = "v2.5.0";
 const canBridgeReleaseAssetUrlPrefix = `https://github.com/REVrobotics/CANBridge/releases/download/${canBridgeTag}`;
 
 const externalCompileTimeDepsPath = 'externalCompileTimeDeps';
-const runtimeArtifactsPath = path.join('prebuilds', 'node_canbridge-win32-x64');
+const runtimeArtifactsPath = {
+    win: path.join('prebuilds', 'node_canbridge-win32-x64'),
+    osx: path.join('prebuilds', 'node_canbridge-darwin-osxuniversal'),
+    linux: path.join('prebuilds', 'node_canbridge-linux-x64'),
+    linuxArm: path.join('prebuilds', 'node_canbridge-linux-arm64'),
+    linuxArm32: path.join('prebuilds', 'node_canbridge-linux-arm32')
+};
 const tempDir = 'temp';
 
 try {
-    await Promise.all(Array.of(
-        downloadCanBridgeArtifact('CANBridge.lib', externalCompileTimeDepsPath),
-        downloadCanBridgeArtifact('CANBridge.dll', runtimeArtifactsPath),
-        downloadCanBridgeArtifact('wpiHal.lib', externalCompileTimeDepsPath),
-        downloadCanBridgeArtifact('wpiHal.dll', runtimeArtifactsPath),
-        downloadCanBridgeArtifact('wpiutil.lib', externalCompileTimeDepsPath),
-        downloadCanBridgeArtifact('wpiutil.dll', runtimeArtifactsPath),
-        downloadCanBridgeArtifact('headers.zip', tempDir),
-    ));
+    // TODO: Do not hardcode the filenames, instead get them from the GitHub API -> Look at Octokit: https://github.com/octokit/octokit.js
```

Comment:
Octokit is great if you need it, but I don't really see the problem with hardcoding URLs here.  We need to hardcode a specific tag, so using Octokit to tell us what the latest version is wouldn't be useful. Octokit could give us a full list of artifacts, but we'd still need to hardcode which ones we want to use (unless we wanted to try to use some sort of heuristics to figure out which file contains what we're looking for, which feels unnecessary). What do you see as the advantages of using Octokit in this context?

Reply:
I _think_ what I was after was to have it always pull from latest and to iterate through every single asset that was in a release, rather than hardcoding the files and the version to be specific on what was needed to be downloaded. Memory on what that was specifically for is a bit hazy. In my mind it would cut down on commits updating the tag as well as not having to worry about missing artifacts if I just download them all from an endpoint.

Little more complex, but would be more extendable as far as what could be available in the future. This note can be totally tossed out if we don't care about those commits or hardcoding. More of a "hey check this out, might help with something in the future" kind of thing.